### DISCUSSION TITLE
Missing jar in latest release

### DISCUSSION THREAD

**Discussion by beatrycze-volk** [dbmdz/solr-ocrhighlighting#196] [General]
Hi, first I would like to thank for fixing error with empty OCR. We would like to try it, but unfortunately the jar file is missing, Could you add this file?

**Comment by jbaiter** [comment]
Ooops, sorry about that! I built packages with OpenJDK8 and attached them to the release just now.

**Reply by beatrycze-volk** [reply]
Thank you a lot!
